importers/ableroad.com/easy_importer.py

```diff
@@ -0,0 +1,226 @@
+    category = getcategory(category)
+    row.append(category)
+
+    distance = dombus.xpath('.//div[@class="itemdistance"]/text()')
+    distance = getdistance(distance)
+    row.append(distance)
+
+    neighborhood = dombus.xpath('.//div[@class="neighborhood"]/text()')
+    neighborhood = getneighborhood(neighborhood)
+    row.append(neighborhood )
+
+    ableroadating = dombus.xpath('.//div[@class="searchableRating"]//div[@class="visually-hidden startableft"]/text()')
+    ableroadating = getableroad(ableroadating)
+    row.append(ableroadating)
+
+    yelprating = dombus.xpath('.//img[@class="yelprating"]/@alt')
+    yelprating = getyelp(yelprating)
+    row.append(yelprating)
+
+    address = dombus.xpath('.//address/text()')
+
+    street = getstreet(address)
+    row.append(street)
+
+    city = getcity(address)
+    row.append(city)
+
+    state = getstate(address)
+    row.append(state)
+
+    postcode = getpostcode(address)
+    row.append(postcode)
+
+    phone = getphone(address)
+    row.append(phone)
+
+    latlng = dombus.getprevious().text #lat lng is hidden in javascript
+    latlng = getlatlng(latlng)
+    row = row + latlng
+    return row
+
+def write_csv(rows):
+    csv_filename = 'export.csv'
+    if os.path.exists(csv_filename):
+        csv_file = open(csv_filename,'a')
+        writer = csv.writer(csv_file, delimiter=',',quotechar='"', quoting=csv.QUOTE_ALL)
+    else:
+        csv_file = open(csv_filename,'w')
+        writer = csv.writer(csv_file, delimiter=',',quotechar='"', quoting=csv.QUOTE_ALL)
+        title = ['name','category','distance','neighborhood','ablerating','yelprating','street','city',
+                'state','postcode','phone','lat','lng']
+        writer.writerow(title)
+    for row in rows:
+        writer.writerow(row)
+    csv_file.close()
+    logging.info('csv file has been written!')
+
+def download_if_not(url, islocalfile):
+    pagecontent = ''
+
+    if islocalfile and os.path.exists('tmp.html'):
+        f = open('tmp.html','r')
+        pagecontent = f.read()
+    else:
+        page = requests.get(url)
+        f = open('tmp.html','w')
+        f.write(page.content)
+        pagecontent = page.content
+
+    f.close()
+    return pagecontent
+
+def gen_url():
+    locations = ['windsor, ontario']
+    for location in locations:
+            for category_id in range(2, 22):
+                for page in range(0,10):
+                    offset = page * 20
+                    url = 'http://ableroad.com/search.php?s=&s1=' + urllib2.quote(location) + \
+                            '&cat=' + str(category_id) + '&offset=' + str(offset) + '&action=search'
+                    logging.info(url)
+                    mainapp(url)
+                    time.sleep(5)
+
+def mainapp(url, islocalfile=False):
+    dom = html.fromstring(download_if_not(url, islocalfile))
+    businesses = dom.xpath('//div[@class="bigresultframe"]')
+    rows = []
+    for bus in businesses:
```

Comment:
Along with renaming businesses to locations, rename bus to location.  "bus" is pretty funny because it is another correctly spelled word so it wouldn't have been a good variable name anyway.